2.0/README.md

```diff
@@ -0,0 +1,137 @@
+Ruby for general usage and OpenShift - Docker images
```

Comment:
Also please make sure those headers are the same in all the READMEs, since when I've was going through all the PR I've noticed that you didn't changed it on all places - eg: https://github.com/openshift/postgresql/pull/71/files